test_analysis/fake_data_stan_analysis.py

```diff
@@ -0,0 +1,448 @@
+#
```

Comment:
It seems to me that this file will quickly become out of sync with the up-to-date version in termite. It would therefore make sense (to me) to simply remove it to avoid confusions...

Reply:
We want to have a version of this script in mermithid as an example/jumping off point for future fake data studies. It is not meant to be kept up-to-date with the version in termite. One option for how to handle this (and what I had in mind to do soon) would be to revise this script so that it applied to a much simpler Stan model, as opposed to being Phase II specific. That way the script would more clearly serve as an example.

I think the same reasoning here applies to  the `p2_analysis` script, since that should also be moved to termite. @Xueying-Huyan